getting-started/templates/systemlink-values.yaml

```diff
@@ -92,11 +92,39 @@ global:
     install: false
   ##
   ## Specify secrets containing trusted certificates that should be installed in all the necessary applications.
-  ## The secrets must be deployed manually prior to installing this chart.
+  ## The secrets must be deployed manually prior to installing this chart. Config maps may be used when it is not possible to use a secret.
   ## Example:
-  ##  - secretName: "S3Certificate"
-  ##    key: cert
+  ##  - secretName: "ca-secret"
+  ##    key: "cert"
+  ##  - secretName: "ca-config-map"
+  ##    key: "my-cert"
+  ##    isConfigMap: true
+  ##
+  #  <ATTENTION> - If one or more custom Certificate Authorities are required for HTTPS access to this application
+  #                or for TLS access to any external resources referenced by this application (databases, file storage, etc.),
+  #                a copy of the certificate chain must be deployed as a secret to the cluster prior to deploying this application.
+  #                For certificate authorities, only the public certificate needs to be deployed to the cluster.
```

Comment:
I don't understand exactly what we are communicating in this line. I could see this meaning one of two things. 

- For deployments using certificates signed by a public certificate authority, administrator need not also deploy the root certificate or certificate chain using this configuration.

OR 

- "For certificate authorities only the root public certificate needs to be deployed to the cluster" e.g. the entire trust chain ins't needed, just the root.

Reply:
Neither. I don't want them to deploy the private key data for the certificate. Just the public certificate.